mh/pat_item.py

```diff
@@ -791,28 +1062,30 @@ def getHunterStats(hr=921, profile=b"Navaldeus",
 
     data = to_bytearray(b"\0" * 0x100)  # fuzz.repeat(fuzz.MSF_PATTERN, 0x100)
 
-    def slot(type_id, equipment_id, slots=0):
+    def slot(type_id, slots, equipment_id, unk1=0, unk2=0):
         """Equipment slot / TODO: Handle gems"""
-        return struct.pack(">BBHII", type_id, slots, equipment_id, 0, 0)
+        return struct.pack(">1+2+2+4+2", type_id, slots, equipment_id, unk1, unk2)
```

Comment:
I haven't never seen this `struct.format` notation with numbers, how does it work?